Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -8,8 +8,11 @@ namespace Microsoft.Dafny;
 
 public class CommonOptionBag {
 
-  public static readonly Option<int> ErrorLimit =
-    new("--error-limit", () => 5, "Set the maximum number of errors to report (0 for unlimited).");
+  public static readonly Option<int> VerificationErrorLimit =
+    new("--error-limit", () => 5, "Set the maximum number of verification errors to report (0 for unlimited).");
+
+  public static readonly Option<Boogie.CoreOptions.VerbosityLevel> Verbosity =
```

Comment:
Let's define our own VerbosityLevel type. I don't think this is the type of thing we want to import from Boogie.